### ISSUE TITLE
Some problem running on Linux

### ISSUE THREAD

**Issue by nonzzz** [SukkaW/memdisk#3]
## TL;DR

I using this library in my project and run test on WSL2(Ubuntu). I check the source code and compared with `Darwin`. On `Darwin` the RAM disk will be destroy by call `destroy.sync` but the `Linux` won't. It only do `umount` not both with `remove` but `Darwin` do them together. So i think we should add a `rm -rf ${root}` for Linux system.

Here is my local task 

Fist run test (pass a password, it's ok)
![image](https://github.com/user-attachments/assets/e9f54b7b-1dbf-4d31-8629-7c295525a2d1)

Next call task

![image](https://github.com/user-attachments/assets/b180ef0f-2967-4fad-8fcd-3efc42df9bc3)

## Others

If i remove the directory by manually and the next call can success.
